tests/test_utils.py

```diff
@@ -0,0 +1,29 @@
+import pytest
```

Comment:
Also see https://github.com/qutebrowser/qutebrowser/blob/v1.3.0/tests/unit/utils/test_utils.py#L632-L638 for my tests, but I think yours are fine as well.